spec/controllers/admin/enterprises_controller_spec.rb

```diff
@@ -112,6 +112,52 @@ module Admin
       end
     end
 
+    describe "edit" do
+      let(:admin_user) { create(:admin_user) }      
+      let(:enterprise1) { create(:distributor_enterprise) }
+      let(:enterprise2) { create(:distributor_enterprise) }
+      let(:manager) { create(:user, enterprise_limit: 10, enterprises: [enterprise2]) }
+      let(:time_zone) { Spree::Zone.create(:name => 'South zone')}
+      let(:shipping_method1) { create(:shipping_method, name: 'Pickup', zone_id: time_zone.id, require_ship_address: true, calculator_type: 'OpenFoodNetwork::Calculator::Weight', distributor_ids: [enterprise1.id]) }
+      let(:shipping_method2) { create(:shipping_method, name: 'Delivery', zone_id: time_zone.id, require_ship_address: true, calculator_type: 'OpenFoodNetwork::Calculator::Weight', distributor_ids: [enterprise2.id]) }
+      let(:payment_method1) { create(:payment_method, :name => 'Online', distributor_ids: [enterprise1.id]) }
+      let(:payment_method2) { create(:payment_method, :name => 'COD', distributor_ids: [enterprise2.id]) }
+      let(:enterprise_fee1) { create(:enterprise_fee, :enterprise_id => enterprise1.id) }
+      let(:enterprise_fee2) { create(:enterprise_fee, :enterprise_id => enterprise2.id) }
+
+      context "as admin" do
+        it "sets required variables" do          
+          allow(controller).to receive_message_chain(:spree_current_user).and_return(admin_user)            
+
+          payment_methods = [payment_method1, payment_method2]
+          shipping_methods = [shipping_method1, shipping_method2]
+          enterprise_fees = [enterprise_fee1]
+
+          spree_get :edit, {:id => enterprise1.permalink} 
```

Comment:
It's been long since Ruby introduced the Hash symbol syntax and we prefere that one, instead of hash rockets.

Reply:
@yasirazgar I created a PR to automate this style-related review in the future. Make sure you take a look https://github.com/openfoodfoundation/openfoodnetwork/pull/2045